core/application/impl/app_configuration_impl.hpp

```diff
@@ -148,6 +150,9 @@ namespace kagome::application {
     RuntimeExecutionMethod runtimeExecMethod() const override {
       return runtime_exec_method_;
     }
+    bool useWavmCache() const override {
```

Comment:
I request you to add one more flag (e.g. with name `--purge-wavm-cache`), which would remove all existing runtime caches during application startup. 

This flag should not anyhow interfere with the already introduced one and both of them could be passed at the same time.